framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -169,7 +170,21 @@ public static ConnectionPool getConnectionPool() {
         boolean foundTests = false;
         for (String testGroup : testGroups) {
           if (categories != null && !categories.contains(testGroup)) {
-            continue;
+            if (categories.contains("smoke") &&
+                testGroup.equals("smoke_simba_jdbc") &&
+                !testDefFile.getName().endsWith("apache.json") &&
+                !testDefFile.getName().endsWith("simba.jdbc.json")) {
+              // this json file applies to all Drivers.
+              // this json file is used for smoke tests
+              // this json file has not been modified to include
+              //    smoke_simba_jdbc tests
+              // therefore assume this json file should be modified to include
+              //    smoke_simba_jdbc tests
+              // therefore check for smoke_simba_jdbc tests
```

Comment:
Can we please clean-up the comments? And include clear details on each of the test definition file types (apache.json, simba.jdbc.json, etc.)? 

Also, may be consider renaming apache.json to either "apache.jdbc.json" or just the default ".json" to maintain backward compatibility.